CITATION.cff

```diff
@@ -0,0 +1,21 @@
+# YAML 1.2
+---
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.2.0
+title: 'Scikit-tree: Modern decision-trees compatible with scikit-learn in Python.'
+abstract: 'scikit-tree is a scikit-learn compatible API for building state-of-the-art decision trees. These include unsupervised trees, oblique trees, uncertainty trees, quantile trees and causal trees.'
+authors:
+    - given-names: Adam
+      family-names: Li
+      affiliation: 'Department of Computer Science, Columbia University, New York, NY, USA'
+      orcid: 'https://orcid.org/0000-0001-8421-365X'
```

Comment:
CC: @PSSF23 @SUKI-O @sampan501 please push a commit, or comment w/ what should be added for your name.